<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>copgui</class>
 <widget class="QMainWindow" name="copgui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1218</width>
    <height>559</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Center of Pressure </string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>picture/LOGO B7.png</normaloff>picture/LOGO B7.png</iconset>
  </property>
  <property name="statusTip">
   <string>DonnyPH @2022</string>
  </property>
  <property name="whatsThis">
   <string>Center of Pressure</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color : rgb(186, 189, 182)</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>321</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <family>Carlito</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Received Data :</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QTextEdit" name="textEdit">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>120</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Carlito</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>160</y>
      <width>321</width>
      <height>311</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_15">
         <property name="font">
          <font>
           <family>Carlito</family>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Coordinate :</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <layout class="QFormLayout" name="formLayout_6">
           <item row="0" column="1">
            <widget class="QLabel" name="label_6">
             <property name="font">
              <font>
               <family>Carlito</family>
               <pointsize>13</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Left X</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QTextBrowser" name="textBrowser">
             <property name="maximumSize">
              <size>
               <width>150</width>
               <height>60</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Carlito</family>
               <pointsize>15</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="label_7">
             <property name="font">
              <font>
               <family>Carlito</family>
               <pointsize>13</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Right X</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QTextBrowser" name="textBrowser_2">
             <property name="maximumSize">
              <size>
               <width>150</width>
               <height>60</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Carlito</family>
               <pointsize>15</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLabel" name="label_8">
             <property name="font">
              <font>
               <family>Carlito</family>
               <pointsize>13</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Sum (L+R) X </string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QTextBrowser" name="textBrowser_3">
             <property name="maximumSize">
              <size>
               <width>150</width>
               <height>60</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Carlito</family>
               <pointsize>15</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QFormLayout" name="formLayout_7">
           <item row="1" column="1">
            <widget class="QTextBrowser" name="textBrowser_4">
             <property name="maximumSize">
              <size>
               <width>150</width>
               <height>60</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Carlito</family>
               <pointsize>15</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QTextBrowser" name="textBrowser_5">
             <property name="maximumSize">
              <size>
               <width>150</width>
               <height>60</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Carlito</family>
               <pointsize>15</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QTextBrowser" name="textBrowser_6">
             <property name="maximumSize">
              <size>
               <width>150</width>
               <height>60</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Carlito</family>
               <pointsize>15</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLabel" name="label_14">
             <property name="font">
              <font>
               <family>Carlito</family>
               <pointsize>13</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Sum (L+R) Y</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="label_13">
             <property name="font">
              <font>
               <family>Carlito</family>
               <pointsize>13</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Right Y</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="label_12">
             <property name="font">
              <font>
               <family>Carlito</family>
               <pointsize>13</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Left Y</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>10</y>
      <width>354</width>
      <height>32</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_9">
       <property name="font">
        <font>
         <family>Carlito</family>
         <pointsize>19</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Center of Pressure Map :</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="image35">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>50</y>
      <width>511</width>
      <height>421</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>image/map 35 mm big box.jpg</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="image40">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>50</y>
      <width>521</width>
      <height>421</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>image/map 40 mm big box.jpg</pixmap>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>500</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QPushButton" name="quitbutton">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Carlito</family>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>EXIT</string>
       </property>
       <property name="icon">
        <iconset theme="exit">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>279</width>
      <height>549</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="portlabel">
         <property name="font">
          <font>
           <family>Carlito</family>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="text">
          <string>PORT Name</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="baudratelabel">
         <property name="font">
          <font>
           <family>Carlito</family>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Baud rate</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_16">
         <property name="font">
          <font>
           <family>Carlito</family>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Set DX Input : </string>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="2">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Connection Option :</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="connectButton">
           <property name="font">
            <font>
             <family>Carlito</family>
             <pointsize>11</pointsize>
             <weight>75</weight>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>CONNECT</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>image/connect-icon.png</normaloff>image/connect-icon.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="disconnectButton">
           <property name="font">
            <font>
             <family>Carlito</family>
             <weight>75</weight>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>DISCONNECT</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>image/disconnect-icon.png</normaloff>image/disconnect-icon.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="9" column="0" colspan="2">
        <layout class="QFormLayout" name="formLayout_2">
         <item row="1" column="0" colspan="2">
          <widget class="QTextEdit" name="textEdit_2">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>60</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Carlito</family>
             <pointsize>16</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <family>Carlito</family>
             <pointsize>13</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Port Status :</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <family>Carlito</family>
             <pointsize>13</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Not Connected</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="10" column="0" colspan="2">
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <family>Carlito</family>
           <pointsize>13</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Connection Percentage Status:</string>
         </property>
        </widget>
       </item>
       <item row="11" column="0" colspan="2">
        <widget class="QProgressBar" name="progressBar">
         <property name="font">
          <font>
           <family>Carlito</family>
           <weight>75</weight>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>in milimeter</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <widget class="QRadioButton" name="button35">
         <property name="font">
          <font>
           <family>Carlito</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>35 mm</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QRadioButton" name="button40">
         <property name="font">
          <font>
           <family>Carlito</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>40 mm</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QRadioButton" name="button45">
         <property name="font">
          <font>
           <family>Carlito</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>45 mm</string>
         </property>
        </widget>
       </item>
       <item row="12" column="0" colspan="2">
        <layout class="QFormLayout" name="formLayout_3">
         <item row="0" column="0">
          <widget class="QLabel" name="label_11">
           <property name="font">
            <font>
             <family>Abyssinica SIL</family>
             <pointsize>17</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>NOTE :</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="label_18">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>image/Lcenter.png</pixmap>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_19">
           <property name="font">
            <font>
             <family>Abyssinica SIL</family>
             <pointsize>15</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Right Pad </string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_17">
           <property name="font">
            <font>
             <family>Abyssinica SIL</family>
             <pointsize>15</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Left Pad      </string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="label_20">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>image/Rcenter.png</pixmap>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_21">
           <property name="font">
            <font>
             <family>Abyssinica SIL</family>
             <pointsize>15</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Sum Pad</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="label_22">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>image/Scenter.png</pixmap>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEdit">
         <property name="font">
          <font>
           <family>Carlito</family>
           <pointsize>13</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEdit_2">
         <property name="font">
          <font>
           <family>Carlito</family>
           <pointsize>13</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="image45">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>50</y>
      <width>531</width>
      <height>421</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>image/map 45mm big box.jpg</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="Lcenter">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>412</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>image/Lcenter.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="Rcenter">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>412</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>image/Rcenter.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="Scenter">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>412</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>image/Scenter.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="textnoimage">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>210</y>
      <width>301</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Carlito</family>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Set DX Input to load map</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>490</y>
      <width>317</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="font">
        <font>
         <family>Carlito</family>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>CLEAR</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_4">
       <property name="font">
        <font>
         <family>Carlito</family>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>TARE</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>textnoimage</zorder>
   <zorder>image45</zorder>
   <zorder>gridLayoutWidget</zorder>
   <zorder>verticalLayoutWidget_2</zorder>
   <zorder>verticalLayoutWidget_3</zorder>
   <zorder>horizontalLayoutWidget_2</zorder>
   <zorder>verticalLayoutWidget</zorder>
   <zorder>image40</zorder>
   <zorder>image35</zorder>
   <zorder>Lcenter</zorder>
   <zorder>Rcenter</zorder>
   <zorder>Scenter</zorder>
   <zorder>layoutWidget</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
